NitroxServer/GameLogic/Players/PlayerData.cs

```diff
@@ -244,6 +266,13 @@ public PersistedPlayerData(string playerName, ushort playerId)
                 PlayerName = playerName;
                 PlayerId = playerId;
             }
```

Comment:
Same here, can you reuse the larger constructor? `: this(playerName, playerId, null)`.

Reply:
Doing that would also make the "SetPlayerNitroxID" obsolete as the player is created with the ID when calling "GetOrCreatePersistedPlayerData"

Reply:
And that would also mean the PlayerID cant be "null" in PlayerManager and that would make the If statement obsolete